docs/vz330_mellow/electronics/Touchscreen.md

```diff
@@ -3,129 +3,144 @@ layout: default
+    ![Assembly2](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/protective-stickers-display.jpg)
 
-![Assembly4](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly4.jpg)
+2. Place the assembled LCD display in to the front cover
 
-Time to mount it to the Machine:
+    Take care to align the top row of buttons with the provisioned openings in the top of the front cover
 
-![Assembly5](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly5.jpg)
+    ![Assembly1](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly1.jpg)
 
-Installing the DSI cable to the Pi.
+3. Mount the back cover using 4 M3 x 6mm screws
 
-{: .highlight }
-> &#8226;Open the black clip and insert the DSI cable with the exposed contacts to the white part of the connector and close the black clip.
+    Take care to respect the orientation of the back cover. The notches in the back cover should align with the posts on the front cover.  
+    The notched opening in the back cover should be over the ribbon connector of the Core board.
 
-![PI_DSI](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Pi_DSI.jpg)
+    ![Assembly3](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly3.jpg)
 
+4. Insert the 2 M4 x 8mm screws into the mounting hinge, and loosely apply 2 M4 T-nuts to these screws  
 
-## Klipperscreen
+    ![Assembly4](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly4.jpg)
 
-We're going to install Klipperscreen so we can use our touchscreen.
+5. Fasten the mounting hinge to the printer frame using the 2 M4 screws and corresponding T-nuts previously inserted.
 
-First we're going to SSH into the Pi with Putty and we're going to install KIAUH
+    These screws don't need to be overly tight, as you may need to adjust the display to ensure smooth cooperation with the enclosure panels.
 
-```bash
-sudo apt-get update && sudo apt-get install git -y
-```
-```bash
-cd ~ && git clone https://github.com/dw-0/kiauh.git
-```
-```bash
-./kiauh/kiauh.sh
-```
+6. Loosely attach the display assembly to the mounting hinge using 1 M4 x 25mm screw and one M4 hex nut
 
-Then Press 1 for install and hit enter
+    Don't overtighten these screws. The screw acts as a hinge and needs room to move ever so slightly.
 
-![kiauh](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/KIAUH.PNG)
+    If you have not wired up the DSI cable yet, then dismount the display from the mounting hinge again after confirming the hinge works correctly.
 
-Press 5 for Klipperscreen and hit enter. This might take a while to install.
+    ![Assembly5](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly5.jpg)
 
-![klipperscreen](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Option5.PNG)
```

Comment:
This section has to stay in since not everyone uses Mainsail.

Reply:
We did put the instructions to do exactly that in to the Firmware installation.

We have to choose whether we do or don't support non-standard configurations. 😅

Reply:
in this instance we're keeping this in here. There's also users that build there machines before mainsail had it in stock that would like to know how to install it.